src/Libraries/CoreNodeModelsWpf/Charts/Controls/BarChartControl.xaml.cs

```diff
@@ -1,11 +1,13 @@
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
```

Comment:
I think we can get rid of this reference, since `ToObservableCollection` only require Utilities?

Reply:
Yes will do